src/nbl/video/CVulkanPhysicalDevice.cpp

```diff
@@ -900,32 +905,6 @@ std::unique_ptr<CVulkanPhysicalDevice> CVulkanPhysicalDevice::create(core::smart
         properties.limits.shaderFloat16 = vulkan12Features.shaderFloat16;
         if (!vulkan12Features.shaderInt8)
             return nullptr;
-
-        if (properties.limits.shaderFloat16) {
-            if (!vulkan12Properties.shaderSignedZeroInfNanPreserveFloat16)
-                return nullptr;
-            if (!vulkan12Properties.shaderDenormPreserveFloat16)
-                return nullptr;
-            if (!vulkan12Properties.shaderDenormFlushToZeroFloat16)
-                return nullptr;
-            if (!vulkan12Properties.shaderRoundingModeRTEFloat16)
-                return nullptr;
-            if (!vulkan12Properties.shaderRoundingModeRTZFloat16)
-                return nullptr;
-        }
-
-        if (properties.limits.shaderFloat64) {
-            if (!vulkan12Properties.shaderSignedZeroInfNanPreserveFloat64)
-                return nullptr;
-            if (!vulkan12Properties.shaderDenormPreserveFloat64)
-                return nullptr;
-            if (!vulkan12Properties.shaderDenormFlushToZeroFloat64)
-                return nullptr;
-            if (!vulkan12Properties.shaderRoundingModeRTEFloat64)
-                return nullptr;
-            if (!vulkan12Properties.shaderRoundingModeRTZFloat64)
-                return nullptr;
-        }
```

Comment:
only remove `shaderDenorm` and `shaderRounding` checks

Reply:
we dont know if the other GPUs have issues with that